### ISSUE TITLE
Standone css

### ISSUE THREAD

**Issue by amazingSurge** [alertifyjs/alertify.js#53]
Can we not inject the css with js?

```
if (! hasCss) {
            var head = document.getElementsByTagName("head")[0];
            var css = document.createElement("style");
            css.type = "text/css";
            css.innerHTML = "/* style.css */";
            head.insertBefore(css, head.firstChild);
            hasCss = true;
}
```

It's causing issues when i try style the elements.

**Comment by bradberger** [comment]
Sure, that shouldn't be a problem to add as an option. Will add it to the to-do list for the next release.

**Comment by bradberger** [comment]
In the meantime, also wondering if wouldn't mind explaining the issues you're coming across so we can address those as well. It should be set up in a way that allows you to style the elements afterwards, so if that's not working than it we'll fix it too.

**Comment by bradberger** [comment]
@amazingSurge This just landed in `v1.0.5`, the docs are on the website now. Just basically requires including of any element with an id of "alertifyCSS" before including the javascript.

**Comment by amazingSurge** [comment]
Thanks!
